packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Fixed
```

Comment:
Quick question: Should there be a whitespace here or is this okay for the NPM release script?

Reply:
@kenk2 - I am not aware of an issue.  If there is an issue, let me know, and I will update accordingly.

Reply:
It shouldn't affect the script.